octavia_f5/controller/worker/l2_sync_manager.py

```diff
@@ -250,20 +251,17 @@ def sync_l2_selfips_flow(self, selfips: [network_models.Port], network_id: str,
 
             store = {'bigip': bigip, 'network': network}
             selfips_for_host = [selfip for selfip in selfips if bigip.hostname in selfip.name]
-            fs[self.executor.submit(self._do_sync_l2_selfips_flow,
-                                    expected_selfips=selfips_for_host,
-                                    store=store)] = bigip
-            fs[self.executor.submit(self._do_sync_l2_static_routes_flow,
-                                    selfips=selfips_for_host, store=store)] = bigip
+            fs[self.executor.submit(self._do_sync_l2_selfips_and_subnet_routes_flow,
+                                    expected_selfips=selfips_for_host, store=store)] = bigip
 
         done, not_done = futures.wait(fs, timeout=10)
         for f in done | not_done:
             bigip = fs[f]
             try:
                 f.result(0)
             except Exception as e:
-                self._metric_failed_futures.labels(bigip.hostname, 'sync_l2_selfips_flow').inc()
-                LOG.error("Failed running sync_l2_selfips_flow for host %s: %s", bigip.hostname, e)
+                self._metric_failed_futures.labels(bigip.hostname, 'sync_l2_selfips_and_subnet_routes_flow').inc()
+                LOG.error("Failed running sync_l2_selfips_and_subnet_routes_flow for host %s: %s", bigip.hostname, e)
```

Comment:
If changing metric names keep in mind changing grafana dashboard.

Reply:
If I am not mistaken there is nothing to change on the grafana dashboards. `sync_l2_selfips_and_subnet_routes_flow` is not a metric name, but a value of the `task` label, and it's not explicitly used anywhere on the Octavia grafana dashboard. The Octavia grafana dashboard only includes a `by (...) task` clause.